app/src/main/res/values-de/strings.xml

```diff
@@ -298,4 +298,6 @@
     <string name="noCameraFoundGuideText">Dein Gerät scheint nicht über eine Kamera zu verfügen. Wenn dies der Fall ist, versuche, das Gerät neu zu starten. Andernfalls verwende die Schaltfläche Weitere Optionen unten, um einen Barcode auf andere Weise hinzuzufügen.</string>
     <string name="exportCancelled">Export abgebrochen</string>
     <string name="importCancelled">Import abgebrochen</string>
+    <string name="useFrontImage">Startbild verwenden</string>
```

Comment:
@arbadacarbaYK The front is referred to in other places as `Vorderseite`. Shouldn't this be something similar too instead of "Start"?

Reply:
I changed to not always use the same word which i assumed is not nice to read. Front translates to a lot (and is also not destinct to Frontpage. Landing Page is. Wasnt sure where that ends up also.

Reply:
I think it's quite important to consistently use the same word. If you switch up the words, users will get confused and not understand a thing that's on multiple different screens is the same thing.

This is also not about a front-page or landing page, it's about the front side of a plastic loyalty card.

Reply:
oh see! Thought cards is about tabs.. turns out translating via weblate without context is not always ideal :P 
change to your wish. Since i am also in NFC cards we should connect on TG :D